check_mk_telegram-notify.sh

```diff
@@ -58,9 +58,30 @@ else
         echo "Invalid privacy parameter, check your Check_MK settings." >&2
 fi
 
+# Set an appropriate emoji for the current state
+if [[ ${NOTIFY_WHAT} == "SERVICE" ]]; then
+        STATE="${NOTIFY_SERVICESHORTSTATE}"
+else
+        STATE="${NOTIFY_HOSTSHORTSTATE}"
+fi
+case "${STATE}" in
+    OK|UP)
+        EMOJI="✅ "
+        ;;
+    WARN)
+        EMOJI="⚠ "
+        ;;
+    CRIT|DOWN)
+        EMOJI="🔥 "
+        ;;
+    UNKN)
+        EMOJI="❓ "
+        ;;
+esac
+
 # Create a MESSAGE variable to send to your Telegram bot
 MESSAGE="${NOTIFY_HOSTNAME} (${NOTIFY_HOSTALIAS})%0A"
-MESSAGE+="${NOTIFY_WHAT} ${NOTIFY_NOTIFICATIONTYPE}%0A%0A"
+MESSAGE+="${EMOJI}${NOTIFY_WHAT} ${NOTIFY_NOTIFICATIONTYPE}%0A%0A"
```

Comment:
Personally, I would not generate the space between emoji and text in the variable, but in the code.

MESSAGE+="${EMOJI} ${NOTIFY_WHAT} ${NOTIFY_NOTIFICATIONTYPE}%0A%0A"